Bio/codonalign/__init__.py

```diff
@@ -6,19 +6,19 @@
 
 """Code for dealing with Codon Alignments."""
 
+import copy
 from collections.abc import Mapping, Iterable
 
 from Bio import BiopythonWarning
 from Bio import BiopythonExperimentalWarning
 
-from Bio.Alphabet import _get_base_alphabet
 from Bio.SeqRecord import SeqRecord
+from Bio.Data.CodonTable import generic_by_id
+
+default_codon_table = copy.deepcopy(generic_by_id[1])
```

Comment:
Why do we need the copy? Does the code make alterations to the dictionary? If so, won't it then modify ``default_codon_table`` too?

Reply:
I don't know. This was in the code originally (in `codonalphabet.py`, which is now deleted).